backend/src/election/api.rs

```diff
@@ -250,6 +268,11 @@ pub struct ElectionAndCandidatesDefinitionImportRequest {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[schema(value_type = Option<String>, nullable = false)]
     polling_station_data: Option<String>,
+    #[schema(value_type = Option<VoteCountingMethod>, nullable = false)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    counting_method: Option<VoteCountingMethod>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    number_of_voters: Option<u32>,
```

Comment:
This also needs `#[schema(nullable = false)]` to prevent the `| null` in the OpenAPI schema

Reply:
I'm not quite sure why the `skip_serializing_if` annotations were added in this struct? It is never serialized to the frontend anyway? I also don't get the `nullable = false` annotations, the fields are nullable. And it should also work just fine, any of these fields can be null from the APIs point of view, as long as they are not null as soon as you do your final submit, but that is using a separate struct with non-nullable fields.

Reply:
It does end up in the OpenAPI spec and we don't want the to have TypeScript types that can be `undefined` or `null`, but only `undefined`.